external-crates/move/crates/move-compiler/src/unit_test/mod.rs

```diff
@@ -77,7 +75,7 @@ impl TestPlan {
     pub fn new(
         tests: Vec<ModuleTestPlan>,
         files: FilesSourceText,
-        units: Vec<AnnotatedCompiledUnit>,
+        units: Vec<NamedCompiledModule>,
```

Comment:
Main change: we don't need all of `AnnotatedCompiledUnit` here. Callers are updated to map `AnnotatedCompiledUnit` to just `NamedCompiledUnit` needed by `TestPlan::new`.